assets/model_monitoring/amlcompute_components/data_quality/data_quality_signal_monitor_command/spec.yaml

```diff
@@ -59,11 +59,6 @@ jobs:
     outputs:
       signal_metrics:
         type: mltable
-    resources:
-      instance_type: standard_e4s_v3
-      runtime_version: "3.3"
-    identity:
-      type: aml_token
   feature_selection:
     type: spark
     component: azureml://registries/azureml/components/model_monitor_feature_selector_command/versions/0.0.1
```

Comment:
Feature selector was bumped to 0.0.2

Reply:
Good catch!

Reply:
Actually that is data quality monitor. I'd like to skip to avoid adding lots of changes not related to data drift signal. For example, the properties only apply to spark jobs in monitors of other signal types.

Reply:
It should be okay given these components are not in used by anyone (version 0.0.*).